client/components/reader-infinite-stream/index.js

```diff
@@ -8,72 +8,81 @@ import {
 	CellMeasurerCache,
 	CellMeasurer,
 	InfiniteLoader,
-	AutoSizer,
 } from 'react-virtualized';
 import { debounce, noop, get } from 'lodash';
 
 /**
  * Internal Dependencies
  */
-import ConnectedSubscriptionListItem from 'blocks/reader-subscription-list-item/connected';
+import { recordTracksRailcarRender } from 'reader/stats';
 
-// this component is being cleaned up in: https://github.com/Automattic/wp-calypso/pull/14577
 class ReaderInfiniteStream extends Component {
 	static propTypes = {
 		items: PropTypes.array.isRequired,
+		width: PropTypes.number.isRequired,
+		rowRenderer: PropTypes.func.isRequired,
 		fetchNextPage: PropTypes.func,
-		forceRefresh: PropTypes.any, // forceRefresh can be anything. Whenever we want to force a refresh, it should change
+		hasNextPage: PropTypes.func,
 		windowScrollerRef: PropTypes.func,
-		showLastUpdatedDate: PropTypes.bool,
-		followSource: PropTypes.string,
-		itemType: PropTypes.oneOf( [ 'site', 'post' ] ),
+		extraRenderItemProps: PropTypes.object,
+		minHeight: PropTypes.number,
+		renderEventName: PropTypes.string,
+		passthroughProp: PropTypes.any,
```

Comment:
When we change underlying data we need a way to inform rv since we don't actually ever give rv the underlying list that is backing the rowRenderers.  E.g. one piece of data we always give rv is the rowCount -- if rowCount changes, then rv knows to rerender.  

What if the sort changes though? 
Its all best explained here: https://github.com/bvaughn/react-virtualized#pure-components

Our two options are:
1. create a single passthroughProp 
2. create a a passthroughProps object that gets expanded into the list via `{ ..passthroughProps }`